.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# Pull Request Template
+
+## Discord username (optional)
+
+If you're in our Discord server please include your formatted username so we can attribute you with our Contributor Discord role! Please include the #XXXX our regex depends on it: elvis#4747
```

Comment:
how about leaving this in the title itself? Maybe something like:

```
## Discord username (optional, if you're in our discord username please include your username so we can attribute you with our Contributor Discord role)
```

Assuming that's not too long, it will help because users can just enter their username directly without wiping anything away